docker-compose.yaml

```
@@ -0,0 +1,112 @@
+name: opik
+
+services:
+  mysql:
+    image: mysql:8.4.2
+    hostname: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: opik
+      MYSQL_DATABASE: opik
+      MYSQL_USER: opik
+      MYSQL_PASSWORD: opik
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-h", "127.0.0.1", "--silent"]
+      timeout: 1s
+      interval: 1s
+      retries: 300
+    ports:
+      - "3306:3306"
```

```suggestion
      - "3306:3306"
    volumes: # Mounted on your $HOME folder
      - ~/opik/mysql:/var/lib/mysql
```

If the don't specify the volumes, all the mysql data is lost when starting and stopping the service

---

migrations run is already there

---

added volume for mysql